nntrainer/layers/mol_attention_layer.cpp

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Parichay Kapoor <pk.kapoor@samsung.com>
+ *
+ * @file   mol_attention_layer.cpp
+ * @date   11 November 2021
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Parichay Kapoor <pk.kapoor@samsung.com>
+ * @bug    No known bugs except for NYI items
+ * @brief  This is MoL Attention Layer Class for Neural Network
+ *
+ */
+
+#include <mol_attention_layer.h>
+#include <nntrainer_error.h>
+#include <nntrainer_log.h>
+
```

Comment:
please add <layer_context.h>, related to #1724

Reply:
done.